packages/webapp/components/layouts/SettingsLayout/Profile/index.tsx

```diff
@@ -164,231 +176,269 @@ const ProfileIndex = ({
     </span>
   );
 
+  const { onUpload } = useUploadCv({ shouldShowSuccessModal: true });
+
+  const uploadSection = (
+    <AccountContentSection
+      className={{ heading: 'mt-0' }}
+      title="Your next job should apply to you"
+      description="Upload your CV so we can quietly start matching you with roles that actually fit your skills and interests. Nothing is ever shared without your permission, and we’ll only reach out when there’s something genuinely worth your time. No spam, no pressure."
+    >
+      <DragDrop
+        renameFileTo={user.name}
+        className="my-4 max-w-[18.5rem]"
+        onFilesDrop={([file]) => onUpload(file)}
+      />
+      {!user?.flags?.cvUploadedAt ? (
+        <FeelingLazy />
+      ) : (
+        <Typography
+          type={TypographyType.Caption1}
+          color={TypographyColor.Quaternary}
+        >
+          Tip: Complete your profile below to improve match quality
+        </Typography>
+      )}
+    </AccountContentSection>
+  );
+
+  const { shouldShow } = useShowUpload();
+
   const form = (
-    <form ref={formRef} id="submit-profile">
-      <AccountContentSection
-        className={{ heading: 'mt-0' }}
-        title="Profile Picture"
-        description="Upload a picture to make your profile stand out and let people recognize
+    <ConditionalWrapper
+      condition={shouldShow}
+      wrapper={(component) => (
+        <>
+          {uploadSection}
+          {component}
+        </>
```

Comment:
Changes looked massive, but the only change here is this wrapping and the component I combined this with.